src/components/poll/SubmitChoices.tsx

```diff
@@ -31,6 +44,16 @@ const SubmitChoices = (props: {
         };
         const submitChoiceResponse = await markChoices(voterArgs);
         if (submitChoiceResponse.statusCode === 201) {
+          if (isLoggedIn) {
+            const mailerArgs: MailerArgs = {
+              pollID,
+              senderEmailID,
+              pollTitle: pollFromDB.title,
+              receiverIDs: Array<string>(pollCreatorEmailID),
```

Comment:
What does ```Array<string>(pollCreatorEmailID)``` here do?
Is it an operation similar to ```[pollCreatorEmailID]``` ?

Reply:
Yes its the same thing